stories/constants/index.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { withDocs } from 'storybook-readme';
 import { storiesOf } from '@storybook/react';
 import { css } from '@emotion/core';
-
 import * as CONSTANTS from 'src/constants';
 import { Typography } from 'src/shared-components';
+// eslint-disable-next-line import/extensions
 import ConstantsReadme from 'docs/constants.md';
```

Comment:
`md` as been added as known extension so you can remove the `.md` and remove the eslint disable comment here

Reply:
yeah it wouldn't let me commit without it. is it because its not a typescript file?

<img width="348" alt="Screen Shot 2020-05-12 at 1 05 58 PM" src="https://user-images.githubusercontent.com/38407520/81740175-61a9c800-9451-11ea-8d19-3e6e84ddef85.png">